runtime/themes/everforest_dark.toml

```diff
@@ -0,0 +1,93 @@
+# Everforest (Dark Hard)
+# Author: CptPotato
```

Comment:
I don't have an email address that I disclose publicly.
Is it a problem to leave it like that?

Reply:
You can also use your GitHub-provided email which is meant for this kind of purpose, but I think this should be fine too.

Reply:
I think it should be fine if you don't want to disclose your email. As long as we know someone that is using it and we can find out easily should be fine.